### PR TITLE
[Flow EVM] Avoid hex encoding/decoding for event fields of type bytes

### PR DESCRIPTION
**Pull request by turbolent**
Follow-up to #6190

### CODE REVIEW

File: fvm/evm/stdlib/contract.cdc [hunk 1]
```diff
@@ -21,27 +21,27 @@ contract EVM {
         // height or number of the block
         height: UInt64,
         // hash of the block
-        hash: String,
+        hash: [UInt8],
```
Comment (ramtinms) [inline fvm/evm/stdlib/contract.cdc:24]: Maybe [UInt8;32] for hash values?

**Review by ramtinms — APPROVED** [review]

File: fvm/evm/stdlib/contract.cdc (hunk 1 above)
Comment (turbolent) [reply fvm/evm/stdlib/contract.cdc:24]: Good idea! Does this also apply to the other hashes changed in the PR? i.e.:
- `BlockExecuted.parentHash`
- `BlockExecuted.receiptRoot`
- `BlockExecuted.transactionHashes`
- `TransactionExecuted.hash`

Comment (ramtinms) [reply fvm/evm/stdlib/contract.cdc:24]: Yes, also all of those, I suggest skipping BlockExecuted.transactionHashes as I'm going to deprecate it in my PR.

**Review by sideninja — APPROVED** [review]
Nice improvement, thank you.

**Review by janezpodhostnik — APPROVED** [review]

File: fvm/evm/stdlib/contract.cdc (hunk 1 above)
Comment (turbolent) [reply fvm/evm/stdlib/contract.cdc]: Changed in aa8761d77ce39c3c442a9903c3149a0356789966

**Pull request merged**